addons/units/data/uniform_combat_atacs.rvmat

```diff
@@ -0,0 +1,90 @@
+class StageTI
+{
```

Comment:
This file has to be cleaned up to standards.
- Indentation
- Braces position
- Spaces between numbers